sdk/networkcloud/Azure.ResourceManager.NetworkCloud/tests/NetworkCloudManagementTestEnvironment.cs

```diff
@@ -22,6 +23,8 @@ public class NetworkCloudManagementTestEnvironment : TestEnvironment
         public string ClusterExtendedLocation => GetRecordedVariable("CLUSTER_EXTENDED_LOCATION");
         public string ClusterRG => GetRecordedVariable("CLUSTER_RG");
         public string ClusterVersion => GetRecordedVariable("CLUSTER_VERSION");
+        public string UserAssignedIdentity => GetRecordedVariable("USER_ASSIGNED_IDENTITY");
+        public string ContainerUri => GetRecordedVariable("CONTAINER_URI");
```

Comment:
This should be updated to pass the lambda using IsSecret along with what the sanitized value should be.